### DISCUSSION TITLE
How to use your drift data for Android and IOS home screen widgets?

### DISCUSSION THREAD

**Discussion by SungHyun22** [simolus3/drift#2539] [Q&A]
The problem I am facing is that I need to create a home screen widget based on the data from Drift. As a solution to this, I am currently trying to open the sqlite generated by Drift in my Android, IOS native code. I want to create a home screen widget based on the data obtained, is this behavior possible?

If you have any other ideas besides this, please feel free to share! 

Thanks :)

**Comment by simolus3** [comment]
> As a solution to this, I am currently trying to open the sqlite generated by Drift in my Android, IOS native code

That's probably what I would have done as well. It's not like there really is an alternative, right? You can't create these widgets with Flutter (as far as I know), so if you need the database natively, that's the right way to go.
